### ISSUE TITLE
Invalid status when session is not running

### ISSUE THREAD

**Issue by rinigus** [sailfishos-open/waydroid#1]
As observed in Settings, Session status was shown as invalid when it is not active. Not sure whether session status is updated automatically as well as it seems to get stuck in "activating" or similar. Would have to check that again.

**Comment by rinigus** [comment]
On new install, I still have "invalid" status. I can still press "Start daemon" that will start the session but will not reflect it on status nor active button. For that, I need to get out of Waydroid settings and get back there. It will show status as active with corresponding button activated.

Reopening, but see one new issue before starting working on it :)
